Lib/test/test_zstd/test_core.py

```diff
@@ -0,0 +1,2506 @@
+import array
+import gc
+import io
+import pathlib
+import pickle
+import random
+import builtins
+import re
+import os
+import unittest
+import tempfile
+import threading
+
+from compression._common import _streams
+
+from test.support.import_helper import import_module
+from test.support import threading_helper
+from test.support import _1M
+from test.support import Py_GIL_DISABLED
+
+zstd = import_module("compression.zstd")
```

Comment:
Does not seem to be used?

Reply:
This is to skip this module when zstd is not available.